README.md

```diff
@@ -1,15 +1,36 @@
-# Assistive Technology ARIA Experience Assessment Project
+# ARIA and Assistive Technologies (ARIA-AT)
 
-The aim of this project is to facilitate consistently effective rendering of ARIA-enabled UIs by assistive technologies by:
+This repo contains a low-level test suite and harness for assessing the interoperability of AT interpretations of ARIA. These tests are designed to be modular and embeddable in higher-level applications.
 
-1. Developing methodologies for assessing how effectively assistive technologies render ARIA patterns.
-2. Providing a framework for collecting, discussing, and disseminating assessment information.
+## Scope and Method
 
-The [ARIA and Assistive Technologies Community Group](https://www.w3.org/community/aria-at/):
+The initial scope of this test suite is manual tests for screen readers. Tests are written as assertions against accessible design patterns [examples from the ARIA Practices Guide](https://w3c.github.io/aria-practices/examples/).
 
-* Is managed by the [Authoring Practices Task Force](https://www.w3.org/WAI/ARIA/task-forces/practices/) of the [ARIA Working Group](http://www.w3.org/WAI/ARIA/).
-* Staff contact: [Michael Cooper](http://www.w3.org/People/cooper/).
+## Running the tests
 
-Please do not provide commit access to this repository without coordination.
+This repo includes a [prototype runner](https://w3c.github.io/aria-at/runner) which you can use to run the tests manually. Production runs of this material are run with [ARIA-AT-App](https://github.com/bocoup/aria-at-app).
+
+## Documentation
 
 [Documentation is available in the wiki](https://github.com/w3c/aria-at/wiki).
+
+## Get involved
+
+You can participate in the following:
+
+* Join the [community group](https://www.w3.org/community/aria-at/)
+* [Sign up as a tester](https://github.com/w3c/aria-at/issues/162) for the Pilot Test (May 27 - June 2)
```

Comment:
To keep the readme from changing frequently, it might be better to link to a generic 'Sign up as a tester' wiki page, which we update every time we are about to run a new testing cycle. Thoughts?

Reply:
We don't have such a wiki page, and I'd like to land this change asap. We can do that in the next iteration though!